### PR TITLE
change biography jinja filters to correctly handle html and new lines.

### LINKED ISSUE #5356: User bio HTML support broken
See for example this profile: https://addons.mozilla.org/user/MaDonna/

The HTML is just shown directly on the page, rather than parsed. I would guess this is related to #5264.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #5356

### CODE REVIEW

File: src/olympia/users/tests/test_views.py [hunk 1]
```diff
@@ -796,12 +796,21 @@ def test_no_self_abuse(self):
...
+                      u'<b>acceptably bold</b>')
         assert '<script>' in self.user.biography
-        r = self.client.get(self.url)
-        assert '<script>' not in r.content
-        assert '&lt;script&gt;alert("xss")&lt;/script&gt;' in r.content
+        response = self.client.get(self.url)
+        assert '<script>' not in response.content
+        assert 'http://spam.com/' not in response.content
+
+        assert 'alert("xss")' in response.content
```
Comment (diox) [inline src/olympia/users/tests/test_views.py]: can you check that `<script>` is there, but escaped?

Comment (eviljeff) [inline src/olympia/users/tests/test_views.py]: It's not any more.  Unknown tags are cleaned entirely.

Comment (diox) [inline src/olympia/users/tests/test_views.py]: Aaah, I had not noticed that `no_links` was not just doing a `bleach.clean()` but also calling `clean_nl().strip()`. Carry on.

Comment (eviljeff) [inline src/olympia/users/tests/test_views.py]: I was a bit concerned the output would be different than it was when it was a localised field, but jorge seemed to prefer this.

**Pull request merged**